addons/stock/models/stock_inventory.py

```diff
@@ -116,6 +117,12 @@ def _action_done(self):
         self.post_inventory()
         return True
 
+    def action_validate_zqc(self):
+        pickings_to_validate = self.env.context.get('button_validate_picking_ids')
```

Comment:
not sure i get it.. this is called when validating an inventory adjustment right? can't you use the original action_validate and check first if there are any picking_id/s ?

Reply:
It's mostly to keep the same process for stock and stock_barcode. During the barcode flow, we can open multiple times the inventory client action on top of a single picking client actions, so this allows us to stay in the picking validation flow even if we make inventories on both parts